### PR TITLE
Add tag to selected contacts

### PR DESCRIPTION
**Pull request by misstick**
I want to add tags to a contact selection list.

 - I choose 3 contacts,
 - a list of tags appear in contextual menu (radio button list disappear),
 - I can choose one or more tags (select/unselect),
 - those tags are added/removes from selected contacts.
 - I see filter value on top of tags list

then 

 - I click on a selected contact row
 - modal appears
 - I click on 'labels'
 - add/remove/select tag menu appears
 - I add a label : this label is added to contextual menu
 - I remove a label : this label is removed from the contextual menu
 - I select a label : this label is selected into the contextual menu

### Current filter
 - [x] show current filter into contextual Menu
 - [ ] define design 

##### I select some contacts (select:start)
- [x] show contextual menu
- [x] default menu is hidden
- [x] display a list of tags with checkbox
- [x] select/unselect used/unused tags

##### select/unselect some tags (‘tags:select’)
 - [x] save tags selected as a list
 - [x] update tags into contacts

##### select/unselect some contact (‘contacts:select’)
 - [x] update selection into tagContextView 

##### if no selection (select:end)
 - [x] contextual menu disappear

##### Handle mixed-checkbox feature (selected=mixed)
- [ ] if a tag is not checked for all, use ‘mixed’ value
https://www.w3.org/TR/wai-aria/states_and_properties#aria-checked 

##### I add a label
 - [x] this label is added to contextual menu

##### I select a label : 
 - [x] this label is selected into the contextual menu

##### I unselect a label
 - [x] this label is unselected from the contextual menu

## Tests
 - [ ] add test on model
 - [ ] add test on ViewModel
 - [ ] add test on View (events triggered)

### CODE REVIEW

**Comment by frankrousseau** [comment]
@misstick Could you add explanation in the PR description about why you are doing these changes? It makes changes clearer for the community and the reviewers.

**Comment by misstick** [comment]
@frankrousseau : is this better? :)

**Comment by frankrousseau** [comment]
Yes it's better! Next time, don't hesitate to add information about your technical choices.

File: client/app/views/labels/admin.coffee [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    modelEvents:
+        'change:selected': 'updateSelection'
+
+
+    initialize: (args) ->
+        super args...
+        @listenTo @, 'change:tags', @updateSelection
+
+
+    select: (model, models) =>
```
Comment (misstick) [inline client/app/views/labels/admin.coffee]: see here to handle mixed checkbox value : true false, mixed

Comment (m4dz) [inline client/app/views/labels/admin.coffee]: This one is related to https://github.com/cozy/cozy-ui/issues/14. The CSS of the checkbox will be handled by _cozy-ui_, and view'll only need to update the `aria-checked` state for the concerned checkboxes.

**Comment by clochix** [comment]
What is the status of this PR ? A lot of work has been done, is is stable enough to be merged before being lost?

**Comment by frankrousseau** [comment]
@clochix This PR is not finished. We didn't plan to finish it. It's something we'll discuss when the email refactoring will be more advanced.

**Comment by frankrousseau** [comment]
@misstick I think we should do something of this PR. The fact it's open without any change is not good. What do you think we should do about it?

**Comment by m4dz** [comment]
@frankrousseau we still lack a custom checkbox component in Cozy Ui to finish this PR. Maybe we could find some time in next sprint to terminate it properly?

**Comment by misstick** [comment]
@frankrousseau: that should be a pity to loose this usefull feature; I'm agree with @m4dz

**Comment by frankrousseau** [comment]
Yeah removing/losing feature is something hard but we need focus. We'll discuss it with @poupotte at next sprint.

**Comment by m4dz** [comment]
I should agree w/ @frankrousseau. Let's see if we can close it on next sprint. If not, all work won't vaccum, we still be able to re-open the PR later (but if we can finish it quickly, it's definitely more fun :tada: :smile:)

**Comment by m4dz** [comment]
Due to some more works on the codebase, I close this one and pick contents in #278. Thanks @misstick :tada:

**Pull request closed without merging**